stl/src/vector_algorithms.cpp

```diff
@@ -1327,6 +1399,144 @@ namespace {
         }
     }
 
+    // _Minmax has exactly the same signature as the extern "C" functions
+    // (__std_min_Nn, __std_max_Nn, __std_minmax_Nn), up to calling convention.
+    // This makes sure the template specialization is fused with the extern "C" function.
+    // In optimized builds it avoids an extra call, as this function is too large to inline.
+    template <_Min_max_mode _Mode, class _Traits, const bool _Sign>
+    auto __stdcall _Minmax(const void* _First, const void* const _Last) noexcept {
+        using _Ty = std::conditional_t<_Sign, typename _Traits::_Signed_t, typename _Traits::_Unsigned_t>;
+
+        _Ty _Cur_min_val; // initialized in both of branches below
+        _Ty _Cur_max_val; // initialized in both of branches below
+
+        // We don't have unsigned 64-bit stuff, so will use sign correction just for that case
+        constexpr bool _Sign_correction = sizeof(_Ty) == 8 && !_Sign;
+
+#ifndef _M_ARM64EC
+        if (_Byte_length(_First, _Last) >= 16 && _Traits::_Sse_plain_min_max_available()) {
+            const size_t _Sse_byte_size = _Byte_length(_First, _Last) & ~size_t{0xF};
+
+            const void* _Stop_at = _First;
+            _Advance_bytes(_Stop_at, _Sse_byte_size);
+
+            auto _Cur_vals = _Traits::_Load(_First);
+
+            if constexpr (_Sign_correction) {
+                _Cur_vals = _Traits::_Sign_correction(_Traits::_Load(_First), false);
+            }

```

Comment:
This repeats `_Traits::_Load(_First)` but I'm 95% sure we can/should say `_Cur_vals` here. Occurs below on right line 1482.